spec/requests/api/v1/game_play_request_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_helper'
-require './././lib/modules/serviceable'
+require './lib/modules/serviceable'
```

Comment:
I fought this and it took time and playing around to figure out how many levels to jump, I also thought it was too much but seemed to work for me with 3 steps...